CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 - `Faker.Aws.En.s3_action/0` [[@vboivin](https://github.com/vboivin)]
 - `Faker.Aws.Fr.region_name/0` [[@vboivin](https://github.com/vboivin)]
 ### Added
+- `Faker.Airports.icao` [[@brunohkbx](https://github.com/brunohkbx)]
+- `Faker.Airports.iata` [[@brunohkbx](https://github.com/brunohkbx)]
+- `Faker.Airports.En.name` [[@brunohkbx](https://github.com/brunohkbx)]
```

Comment:
Could you please provide function airty?

Reply:
@anthonator done :+1: